packages/editor-preview/package.json

```diff
@@ -4,7 +4,9 @@
   "description": "Theia - Editor Preview Extension",
   "dependencies": {
     "@theia/core": "1.15.0",
-    "@theia/editor": "1.15.0"
+    "@theia/editor": "1.15.0",
+    "@theia/navigator": "1.15.0",
+    "@theia/workspace": "1.15.0"
```

Comment:
@kenneth-marut-work @colin-grant-work is the new coupling necessary?
`@theia/editor` only uses `core`, why does `editor-preview` need the coupling?

Reply:
@vince-fugnitto, I believe that these were added because `editor-preview` now contributes decorations to the open-editors widget that determine whether the workspace is shown as part of the node suffixes and whether the caption should be italicized (preview) or not (not preview).

Reply:
The two of us discussed this since the tree decorator (which is responsible for italicizing the filename when in preview mode, and also adding workspace/tail decorations) was decided to be moved from the navigator -> editor preview package. This allows the decorator to have access to the editor-preview's built-in checks for preview state, and emitters for on-preview-pinned. 

It has the sideeffect of needing to depend on the navigator package (to access the decorator service) as well as the workspace service. The SCM and markers package have a similar dependency set up.